conda_forge_tick/migrators/migration_yaml.py

```diff
@@ -71,6 +104,23 @@ def __init__(
     def migrate(
         self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any
     ) -> "MigrationUidTypedDict":
+        # if conda-forge-pinning do something special
+        if attrs.get("name", "") == "conda-forge-pinning":
+            # read up the conda build config
+            with indir(recipe_dir), open("conda_build_config.yaml") as f:
+                cbc_contents = f.read()
+            merged_cbc = merge_migrator_cbc(self.yaml_contents, cbc_contents)
+            with indir(os.path.join(recipe_dir, "migrations")):
+                os.remove(f"{self.name}.yaml")
+            # replace the conda build config with the merged one
+            with indir(recipe_dir):
+                os.remove("conda_build_config.yaml")
+                with open("conda_build_config.yaml", "w") as f:
+                    f.write(merged_cbc)
+            # don't need to bump build number once we move to datetime
+            # version numbers for pinning
+            return super().migrate(recipe_dir, attrs)
+
         # in case the render is old
         os.makedirs(os.path.join(recipe_dir, "../.ci_support"), exist_ok=True)
```

Comment:
Let's put all of the code here and below in an else statement to make the intent clearer